pkg/cri/resource-manager/cache/utils.go

```diff
@@ -92,33 +83,23 @@ func QuotaToMilliCPU(quota, period int64) int64 {
 
 // sharesToMilliCPU converts CFS CPU shares to milliCPU.
 func sharesToMilliCPU(shares int64) int64 {
-	if shares == minShares {
+	if shares == kubecm.MinShares {
 		return 0
 	}
-	//return int64(float64(shares*milliCPUToCPU) / float64(sharesPerCPU) + 0.5)
-	return int64((float64(shares*milliCPUToCPU) / float64(sharesPerCPU)) + 0.5)
+	return int64(float64(shares*kubecm.MilliCPUToCPU)/float64(kubecm.SharesPerCPU) + 0.5)
 }
 
 // quotaToMilliCPU converts CFS quota and period to milliCPU.
 func quotaToMilliCPU(quota, period int64) int64 {
 	if quota == 0 || period == 0 {
 		return 0
 	}
-	return int64(float64(quota*milliCPUToCPU)/float64(period) + 0.5)
+	return int64(float64(quota*kubecm.MilliCPUToCPU)/float64(period) + 0.5)
```

Comment:
Is `+ 0.5` meant to work as `math.Ceil()` replacement?

Reply:
No, `math.Ceil()` always rounds up, IOW results in the smallest integer larger than or equal to the input. This rounds up or down to the nearest integer instead.

Reply:
Yes, sure, but I'm just concerned about the `0` case. Since we know here that `quota` and `period` are bigger than zero, should we aim to have the milliCPUs to be at least `1`? Other than that the difference of 1 milliCPU is not much, if we are not using this function to handle the "integer CPU" case of figuring out whether or not `milliCPUs % 1000 == 0`.